### DISCUSSION TITLE
Does mosec support all embedding models?

### DISCUSSION THREAD

**Discussion by zengqingfu1442** [mosecorg/mosec#524] [Q&A] [answered]
Does mosec support all embedding models listed here?https://github.com/netease-youdao/BCEmbedding/blob/master/Docs/EvaluationSummary/embedding_eval_summary.md

**Comment by kemingy** [comment, marked as answer]
Mosec supports all the models.

You can use the example code directly if the model can be load with
transformers AutoModel
https://mosec.readthedocs.io/en/latest/examples/embedding.html

Otherwise, you may need to change the code a bit.
